node/coinstacks/solana/api/src/controller.ts

```diff
@@ -126,4 +127,30 @@ export class Solana implements BaseAPI {
       throw handleError(err)
     }
   }
+
+  /**
+   * Get the current recommended gas fees to use in a transaction
+   *
+   * @returns {Promise<number>} current fees specified in lamports including base fee
+   */
+  @Response<BadRequestError>(400, 'Bad Request')
+  @Response<ValidationError>(422, 'Validation Error')
+  @Response<InternalServerError>(500, 'Internal Server Error')
+  @Post('/fees/estimate')
+  async getEstimateFees(@Body() body: GasFeesBody): Promise<number> {
+    try {
+      const deserializedMessage = Message.from(Buffer.from(body.message, 'base64'))
+
+      const feeResult = await heliusSdk.connection.getFeeForMessage(deserializedMessage)
+      const gasPrice = feeResult.value
+
+      if (!gasPrice) {
+        throw new Error('Failed to get gas price')
+      }
+
+      return gasPrice
```

Comment:
destructure and update error to not include gas price vernacular
```suggestion
      const { value } = await heliusSdk.connection.getFeeForMessage(deserializedMessage)

      if (!value) throw new Error('Failed to get estimated fee')

      return value
```